src/SwooleRequestHandlerRunner.php

```diff
@@ -97,14 +98,20 @@ class SwooleRequestHandlerRunner extends RequestHandlerRunner
      */
     private $staticResourceHandler;
 
+    /**
+     * @var string
+     */
+    private $processName;
+
     public function __construct(
         RequestHandlerInterface $handler,
         callable $serverRequestFactory,
         callable $serverRequestErrorResponseGenerator,
         PidManager $pidManager,
         SwooleHttpServer $httpServer,
         StaticResourceHandlerInterface $staticResourceHandler = null,
-        Log\AccessLogInterface $logger = null
+        Log\AccessLogInterface $logger = null,
+        string $processName = null
```

Comment:
Set the default value here to `self::DEFAULT_PROCESS_NAME`, and then you won't need the ternary later.